packages/css/src/button/button.scss

```diff
@@ -14,6 +14,38 @@
   }
 }
 
-.amsterdam-button--secondary:hover {
-  box-shadow: var(--amsterdam-button-secondary-hover-box-shadow);
+.amsterdam-button--secondary {
+  box-shadow: var(--amsterdam-button-secondary-box-shadow);
+
+  &:disabled,
+  [aria-disabled="true"] {
+    box-shadow: var(--amsterdam-button-secondary-disabled-box-shadow);
+  }
+
+  &:hover:not(:disabled, [aria-disabled="true"]) {
+    box-shadow: var(--amsterdam-button-secondary-hover-box-shadow);
+  }
+}
+
+// Specificity hack to be able to style secondary button focus state
+// This isn't currently possible with the Utrecht component
+// We should really pull this component in our own repo...
+.amsterdam-button--secondary.amsterdam-button--secondary {
+  &:focus {
+    color: var(--amsterdam-button-secondary-focus-color);
+  }
+}
+
+.amsterdam-button--secondary:focus:not(:disabled, [aria-disabled="true"]) {
```

Comment:
Should we include `.utrecht-button--disabled` in lists like this?

Reply:
I don't think it's necessary, the Utrecht Button component doesn't set a `box-shadow`, so everything `box-shadow`-related is contained within the `amsterdam-button` namespace